content/docs/lists-and-keys.md

```diff
@@ -202,13 +202,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Experimente no CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+Uma boa regra de definição é que os elementos dentro de `map()` devem especificar chaves.
```

Comment:
Maybe we can use 'regra de ouro' for 'rule of thumb'

Reply:
"via de regra" seems more appropriate. what do you think?

Reply:
also too.

Reply:
So, can I change to this?

```
Por via de regra, os elementos dentro de uma função `map()` devem especificar chaves.
```

Reply:
Yes